hawkular-apm-server/standalone-wrapper.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# where are we located? usually, we are on $JBOSS_HOME/bin , along with the other scripts
+DIRNAME=`dirname "$0"`
+
+# we create only if there's no active users starting with 'admin'
+grep "^admin" "${DIRNAME}/../standalone/configuration/application-users.properties" > /dev/null
+RT=$?
+
+if [ $RT -eq 0 ]; then
+  echo "An admin user has already been found."
+elif [ $RT -ne 1 ]; then
+  echo "An error has been found when attempting to check if an admin user exists. Aborting."
+  exit 1
+else
+  username="admin$(head /dev/random -c 512 | tr -dc A-Z-a-z-0-9 | head -c 7)"
```

Comment:
Good point. However if a suitable solution can be found it would be good.

Maybe as a short term option we could have a 'dev' env var flag that could just set jdoe/password?